### PR TITLE
fixing curl for checking files on pCloudy

### PR DESCRIPTION
**Pull request by nirmalashish**
added a method to set web browser window size

### CODE REVIEW

File: src/main/java/com/znsio/e2e/runner/Setup.java [hunk 1]
```diff
@@ -817,13 +817,18 @@ private CommandLineResponse getListOfUploadedFilesInPCloudy(String authToken) {
...
         payload.put("\"limit\"", 15);
         payload.put("\"filter\"", "\"all\"");
-        String updatedPayload = payload.toString().replace("\"", "\\\"");
+        String updatedPayload = payload.toString().replace("\"", "\\\"").replaceAll("=", ":");
 
-        String[] listOfUploadedFiles;
-        listOfUploadedFiles = new String[]{
-                "curl --insecure -H Content-Type:application/json -d '{\"filter\":\"all\",\"limit\":15,\"token\":\"'" + authToken + "'\"}' '" + deviceLabURL + "/api/drive'\n"};
+        String[] listOfDevices;
+        listOfDevices = new String[]{
```
Comment (anandbagmar) [inline src/main/java/com/znsio/e2e/runner/Setup.java:823]: rename to listOfUploadedFiles

File: src/main/java/com/znsio/e2e/tools/Driver.java [hunk 2]
```diff
@@ -340,4 +340,8 @@ public WebElement waitTillElementIsVisible (String locator) {
     public WebElement waitTillElementIsVisible (String locator, int numberOfSecondsToWait) {
         return (new WebDriverWait(driver, numberOfSecondsToWait)).until(ExpectedConditions.visibilityOf(findElementByAccessibilityId(locator)));
     }
+
+    public void setWindowSize (int width, int height) {
```
Comment (anandbagmar) [inline src/main/java/com/znsio/e2e/tools/Driver.java:344]: where and how will this be used?

**Review by anandbagmar — CHANGES_REQUESTED** [review]

File: src/main/java/com/znsio/e2e/runner/Setup.java (hunk 1 above)
Comment (nirmalashish) [reply src/main/java/com/znsio/e2e/runner/Setup.java]: renamed

File: src/main/java/com/znsio/e2e/tools/Driver.java (hunk 2 above)
Comment (nirmalashish) [reply src/main/java/com/znsio/e2e/tools/Driver.java:344]: there are use cases where a new tab gets opened. For such cases running on CI, we need to set window size. So I thought of creating a method for it.
If needed we can add a check of Driver.type before setting the window size

Comment (anandbagmar) [reply src/main/java/com/znsio/e2e/tools/Driver.java:344]: The check will be good - else it may create problems

Comment (nirmalashish) [reply src/main/java/com/znsio/e2e/tools/Driver.java:344]: added

**Pull request merged**